portaljs-pages/page-admin-case-list/src/index.html

```diff
@@ -10,7 +10,29 @@
             }
         };
 
-        window.location.replace(getPortalUrl() + "/portal.js/#/admin/cases/list");
+        function getParameterByName(name) {
+            var url = window.location.href;
+            name = name.replace(/[\[\]]/g, '\\$&');
```

Comment:
This should be :
`name.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");`
otherwise it is incomplete and only `[` and `]` are escaped (look for escapeRegExp in this page: https://developer.mozilla.org/fr/docs/Web/JavaScript/Guide/Expressions_r%C3%A9guli%C3%A8res)

Reply:
... or we can remove the line as we don't pass any special characters